app/controllers/api/articles_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Api::ArticlesController < ApplicationController
+  def index
+    @article = Article.all
+    render json: @article
```

Comment:
Ni need to use an instance variable here, and also, I usually call on the Serializer. Are you actually using it?

Reply:
Yes we are using it , it is automatically serializing it we believe
We have a file in initializers folder with the following line in it
```
ActiveModelSerializers.config.adapter = :json
```

Reply:
Instance variable is removed by the way